### ISSUE TITLE
Only the 3.0.1 version is possible to install from PyPI.

### ISSUE THREAD

**Issue by BertKiv** [shuttleai/ShuttleASGI#1]
The versions: 3.0.4, 3.0.5 and 3.0.6 throws errors as below:

```
poetry install --no-root
Installing dependencies from lock file

Package operations: 1 install, 0 updates, 0 removals

  - Installing shuttleasgi (3.0.6): Failed

  RuntimeError

  Unable to find installation candidates for shuttleasgi (3.0.6)

  at /usr/local/Cellar/poetry/1.8.2/libexec/lib/python3.12/site-packages/poetry/installation/chooser.py:74 in choose_for
       70│ 
       71│             links.append(link)
       72│ 
       73│         if not links:
    →  74│             raise RuntimeError(f"Unable to find installation candidates for {package}")
       75│ 
       76│         # Get the best link
       77│         chosen = max(links, key=lambda link: self._sort_key(package, link))
       78│ 

Cannot install shuttleasgi.
```

**Comment by herumes** [comment]
How is it with 3.0.7? Same issue?

**Comment by herumes** [comment]
@BertKiv Issues resolved, try with newest versions >=4.0.0
